sql_scripts/populate_album.sql

```diff
@@ -0,0 +1,13 @@
+COPY temp_album
+    FROM '/Users/ramprasad/spotify_dataset_project/spotify_dataset/temp/album.csv'
+    DELIMITER E','
+    CSV HEADER;
+
+SELECT DISTINCT temp_album.album_uri, temp_album.album_name
+INTO album
+FROM temp_album;
+
+ALTER TABLE album
+ADD PRIMARY KEY (album_uri);
```

Comment:
Primary key will be a SERIAL INTEGER value. So add an "id" column and make it primary key.

Reply:
added id column made it serial. Made id as primary key.